index.html

```diff
@@ -0,0 +1,627 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style/style.css" />
+    <title>Home / X</title>
+  </head>
+  <body>
+    <!------ Grid container------>
+    <div class="container">
+      <!------ left-hand menu------>
+      <nav class="left-hand-menu">
```

Comment:
These comments are redundant, your css classes semantic nav element clearly describe the purpose of the content